modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc

```diff
@@ -0,0 +1,33 @@
+= Marketplace Consumption Metrics
+:description: Learn about the consumption units in Redpanda Cloud billing.
+
+When you subscribe to Redpanda Cloud through Google Cloud Marketplace, you can use your existing GCP billing and credits to provision Redpanda Dedicated Cloud clusters. View your bills and manage your subscription directly in Google Cloud Marketplace. You can either pay as you go or purchase through committed use. 
+
+== Redpanda consumption units
+
+Redpanda Cloud billing is mapped to the consumption of resources in your organization, including data ingress and egress and storage usage. To calculate the bill, the number of units consumed is multiplied by the cost for that unit. 
+
+* All pricing is set in US dollars (USD). GCP automatically converts the bill to the currency of the billing account. 
+* All billing computations are conducted in Coordinated Universal Time (UTC). Billing accrues at hourly intervals. Any usage that is less than an hour is billed for the full hour. 
+
+=== Instance meters
+
+Each instance unit equals one hour the instance is up and running. The cost for an instance unit depends on the region and tier you selected for your Dedicated cluster.  
+
+=== Ingress and egress meters
+
+Redpanda Cloud charges for all data transferred each hour, including each produce and consume request.
+
+* Each ingress unit equals one GB/hour of data written. The cost for an ingress unit depends on the region you selected for your cluster. 
+* Each egress unit equals one GB/hour of data read. The cost for an egress unit depends on the number of availability zones (AZ) you selected for your cluster. 
+
+=== Storage meters
+
+Each storage unit equals one GB of data in cloud storage. Redpanda Cloud charges for the volume of data stored on a cluster, including replication. 
```

Comment:
oh is it true? if you have the data with replication=3 we will store it 3 times in S3? I was expecting some deduplication but that said - I don't know :)

Reply:
I think this really is not true - reference https://redpandadata.slack.com/archives/C01ND4SVB6Z/p1694511963692429